src/Card.jsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react'
+import './Card.css'
+
+export default function Card(title, text, isBookmarked) {
+  return (
+    <section className="Card">
+      <div className={isBookmarked ? 'Card bookmark' : 'Card'} />
```

Comment:
You need to add the `bookmark_added` class if it is added, like an active class?